crates/bdk/src/wallet/coin_selection.rs

```diff
@@ -489,7 +491,7 @@ impl CoinSelectionAlgorithm for BranchAndBoundCoinSelection {
                 curr_value,
                 curr_available_value,
                 target_amount,
-                cost_of_change,
+                cost_of_change as f32,
```

Comment:
can the signature of `BranchAndBoundCoinSelection::bnb` change to accept `cost_of_change: u64` as parameter ?